snuba/cli/bulk_load.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+import click
+
+from snuba import settings
+from snuba.datasets.factory import get_dataset
+from snuba.snapshots.postgres_snapshot import PostgresSnapshot
+
+
+@click.command()
+@click.option('--dataset', type=click.Choice(['events', 'groupedmessage', 'outcomes']),
+              help='The dataset to bulk load')
```

Comment:
Shouldn't this list be generated dynamically? Also, why are `events` and `outcomes` in here?

Reply:
Yes it should be generated dynamically. I forgot to do it before. I can introduce it here.